src/components/dialogHelper/dialoghelper.css

```diff
@@ -139,8 +139,8 @@
     }
 
     .dialog-small {
-        width: 60%;
-        height: 80%;
+        width: 100%;
```

Comment:
Something seems a little bit weird here, `.dialog-small` becomes bigger than `.dialog-medium`. Maybe you should use another class or rename this one?